Tests/WorkingWithBrowser.robot

```diff
@@ -0,0 +1,37 @@
+# robot -d  results Tests/WorkingWithBrowser.robot
+*** Settings ***
+Library  SeleniumLibrary
+
+*** Variables ***
+
+
+*** Test Cases ***
+TC to demonstrate Browser Operation Keywords in Robot Framework
+    [Documentation]  TC to demonstrate Browser Operation Keywords in Robot Framework
+
+    Open Browser  http://google.com  Chrome  alias=ChromeRCV
+
+    Maximize Browser Window
+
+    Open Browser  about:blank  ff  alias=RCVFF
+
+    &{alias}  Get Browser Aliases
+
+    Log  @{alias}[1]
```

Comment:
&{alias} with get the aliases ChromeRCV and RCVFF and have them stored like a list - [ChromeRCV,RCVFF]. You'll be able to see this after running in the report under the Log command